uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/setting/Setting.kt

```diff
@@ -0,0 +1,171 @@
+package io.github.droidkaigi.feeder.setting
+import android.content.Context
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.feeder.Theme
+import io.github.droidkaigi.feeder.core.theme.getTitle
+import io.github.droidkaigi.feeder.core.theme.typography
+import io.github.droidkaigi.feeder.core.use
+
+@Preview
+@Composable
+fun Settings() {
+    ProvideSettingViewModel(viewModel = settingViewModel()) {
+        val (
+            state,
+            _,
+            dispatch,
+        ) = use(settingViewModel())
+        val context = LocalContext.current
+
+        Surface(
+            color = MaterialTheme.colors.background,
+        ) {
+            Theme(
+                context = context,
+                theme = state.theme,
+                onClick = {
+                    dispatch(SettingViewModel.Event.ChangeTheme(theme = it))
+                }
+            )
+        }
+    }
+}
+
+@Composable
+fun Theme(
```

Comment:
How about making it private and naming it ThemeSetting, as it can be mistaken for setting Theme? 🎨

Reply:
Thanks, I've changed it that way!